content/general/concepts/data-types/data-types.md

```diff
@@ -21,9 +21,9 @@ In computer programming, primitive data types are the building blocks for any va
 <!-- prettier-ignore -->
 | Data Type | Description |
 | --- | --- |
-| Number | Numbers include `-1`, `0`, `1`, and so on. Depending on the language, numbers with decimals are of a different type. |
-| String | Strings are collections of characters text. Depending on the language, single quotes (`''`) or double quotes (`""`) can be used. For example: `"Codecademy is awesome! 🖥️⌨️"` |
-| Boolean | Booleans hold a `true` or `false` value. |
+| [Number](https://www.codecademy.com/resources/docs/general/number) | Numbers include `-1`, `0`, `1`, and so on. Depending on the language, numbers with decimals are of a different type. |
```

Comment:
For these table rows, I recommend changing the text in the "Description" column to _**not**_ begin with the same word(s) as the word(s) in the previous "Data Type" column. For example, in this line, we should change this to:

```
| [Number](https://www.codecademy.com/resources/docs/general/number) | Includes `-1`, `0`, `1`, and so on. Depending on the language, numbers with decimals are of a different type. |
```

or this:

```
| [String](https://www.codecademy.com/resources/docs/general/string) | Sequences of characters surrounded by single quotes (''), double quotes (""), and/or backticks (\`\`), depending on the language. For example: `"Codecademy is awesome! 🖥️⌨️"` |
```

This also helps our workaround by making the markdown tables look and act more like terms tables

Reply:
Can we make this change for all the markdown tables in this file?

Reply:
Changed it. But Im not sure what to do for boolean and array.

Reply:
@Dusch4593 ^^